build/root/WORKSPACE

```diff
@@ -66,3 +50,10 @@ docker_pull(
     registry = "gcr.io",
     repository = "google-containers/debian-iptables-amd64",
 )
+
+docker_pull(
+    name = "official_busybox",
+    registry = "index.docker.io",
+    repository = "library/busybox",
+    tag = "latest",  # TODO: more hermetic?
```

Comment:
updated to use a digest instead. not sure how to write a test. I guess we can have something that does a docker pull and compares against the digest in the workspace?